Problem 4:
Grades of the final examination in a training course are found to be normally distributed, with a mean of 77 and a standard deviation of 8.5. Based on the given information answer the questions below.


In [1]:
# import the important packages
import pandas as pd 
import numpy as np 
import matplotlib.pyplot as plt 
import seaborn as sns  
%matplotlib inline
import scipy.stats as stats  
from scipy.stats import norm
from scipy.stats import   ttest_1samp , ttest_ind
from statsmodels.formula.api import ols      # For n-way ANOVA
from statsmodels.stats.anova import _get_covariance,anova_lm # For n-way ANOVA
import statsmodels.stats.api as sm

4.1 What is the probability that a randomly chosen student gets a grade below 85 on this exam?

In [131]:
#μ=77
#σ=8.5
round(stats.norm.cdf(85,loc=77,scale=8.5), 4)

0.8267

82.66% of student gets a grade below 85 on this exam

4.2 What is the probability that a randomly selected student scores between 65 and 87?

In [132]:
stats.norm.cdf(87,77,8.5)-stats.norm.cdf(65,77,8.5)

0.8012869336779058

80.12% is the probability that a randomly selected student scores between 65 and 87.

4.3 What should be the passing cut-off so that 75% of the students clear the exam?

In [133]:
round(stats.norm.ppf(0.75,77,8.5),4)

82.7332

82.73 marks is the passing cut-off so that 75% of the students clear the exam.